integrated_channels/blackboard/views.py

```diff
@@ -109,8 +109,19 @@ def get(self, request, *args, **kwargs):  # pylint: disable=arguments-differ
         try:
             data = auth_response.json()
             refresh_token = data['refresh_token']
-        except (KeyError, ValueError):
-            raise requests.RequestException(response=auth_response)
+        except KeyError as exception:
+            raise ParseError(
+                "BLACKBOARD: failed to find refresh_token in auth response. "
+                "Auth response text: {}, Response code: {}, JSON response: {}".format(
+                    auth_response.text,
+                    auth_response.status_code,
+                    data,
```

Comment:
just a note: even though this line will contain refresh token I think we are good security wise since that refresh token only is usable once and gets overwritten in future runs of client ...